server/controllers/users.js

```diff
@@ -171,3 +171,18 @@ export function getUserCampaigns(req, res, next) {
     })
     .catch(err => console.log('could not retrieve user\'s campaigns:', err));
 }
+
+export function getUserCampaignAssociation(req, res) {
+  const params = req.params;
```

Comment:
I like filtering params on both the controller and service, but I'm not sure how necessary that is.